src/util/rotary.cpp

```diff
@@ -1,93 +1,24 @@
 #include "util/rotary.h"
 
-#include <QtDebug>
-
-constexpr int kiRotaryFilterMaxLen = 50;
-
-Rotary::Rotary()
-        : m_iFilterLength(kiRotaryFilterMaxLen),
-          m_iFilterPos(0),
-          m_pFilter(m_iFilterLength),
-          m_dCalibration(1.0),
-          m_dLastValue(0.0),
-          m_iCalibrationCount(0) {
-    for (int i = 0; i < m_iFilterLength; ++i) {
-        m_pFilter[i] = 0.;
```

Comment:
Without that loop, the jog wheel is unfiltered for the first samples. This here assumes that the jog is not moving before the controller is initalized. Is it it a problem? The workarund was the removed fillBuffer() function.

Reply:
I'm not sure. If the loop is pre-filled with zeros then the mean is heavily distorted until those zeros have been flushed.

Reply:
Actually, thinking about this more, this could be the reason why I have had the issue for years where moving the jog suddenly would be slow at first and then really fast after a couple split seconds. I will do some testing to confirm, but now I'm fairly certain that filling the buffer is wrong!